src/lib/action/entry-derive.ts

```diff
@@ -27,40 +27,68 @@ class EntryDeriveAction extends APIAction {
   async applyTo (api: OfflineAPI) {
     const entries: Entry[] = await api.getEntriesForContentType(this.contentTypeId)
     const locales: string[] = await api.getLocalesForSpace()
+
     for (const entry of entries) {
       const inputs = _.pick(entry.fields, this.fromFields)
       const newEntryId = await this.identityKey(inputs)
       const hasEntry = await api.hasEntry(newEntryId)
 
+      let skipEntry = true
+      let fieldsForTargetEntry = {}
+
+      for (const locale of locales) {
+        let outputsForCurrentLocale
+        try {
+          outputsForCurrentLocale = await this.deriveEntryForLocale(inputs, locale)
+        } catch (err) {
+          await api.recordRuntimeError(err)
+          continue
+        }
+
+        if (typeof outputsForCurrentLocale === 'undefined') {
```

Comment:
why this and not just `outputsForCurrentLocale === undefined`?

Reply:
Because `undefined` could be overwritten (treated as variable), so you'd need to `outputsForCurrentLocale === void 0` here :trollface:

Reply:
No good reason :innocent:

Reply:
🤷‍♂️ 

From what I understand TS transpiles by default to JS in strict mode so I would simplify and use the `outputsForCurrentLocale === undefined`